org.chromium.Chromium.yaml

```diff
@@ -149,12 +149,14 @@ modules:
     sources:
       - type: git
         url: https://github.com/refi64/flextop
-        commit: 485c332c7ec36a8b74be8f3622fc756ac6263f40
+        commit: 9f66dac84388f520ecab5ac4fa60df6343b7db5b
 
   - name: extensions
     buildsystem: simple
     build-commands:
       - mkdir -p /app/chromium/{extensions,native-messaging-hosts,policies}
+      - for dir in native-messaging-hosts policies; do ln -s /app/chromium/{$dir,extensions}/$dir; done
```

Comment:
If you `mkdir` dirs other than `extensions` prior to creating symlink, you'll end up with symlinks like
`/app/chromium/extensions/policies/policies` -> `/app/chromium/policies`
I guess this is incorrect?
In any case, I suggest using `ln -s -T` to make sure that the symlink either ends up exactly where you told it, or the command fails.

Reply:
Oh nevermind, I've misread the `{}` expansion.